contributor-agreement-corporate.txt

```diff
@@ -21,7 +21,7 @@
    and to grant copyright and patent licenses thereto.
 
    If you have not already done so, please complete and sign, then
-   scan and email a pdf file of this Agreement to clas@fasterxml.com.
+   scan and email a pdf file of this Agreement to cla@fasterxml.com.
```

Comment:
Might be overkill... but 
```suggestion
   scan and email a pdf file of this Agreement to `cla@fasterxml.com`.
```

Reply:
I don't think this file is markdown

Reply:
Yeah, but still made suggestion thinking it's easier copy-paste with backticks.
No biggie, cool without it 👍🏼